schemas/Manual.items.schema.json

```
@@ -26,76 +26,78 @@
+                    "description": "(Optional) Should this item not get included in progression balance swaps? For more information on item classifications, see: https://github.com/ArchipelagoMW/Archipelago/blob/main/docs/world%20api.md#items",
                     "type": "boolean",
-                    "default": true
+                    "default": false
+                },
+                "useful": {
+                    "description": "(Optional) Is this item useful to have but not required to complete the game? For more information on item classifications, see: https://github.com/ArchipelagoMW/Archipelago/blob/main/docs/world%20api.md#items",
+                    "type": "boolean",
+                    "default": false
                 },
                 "trap": {
-                    "description": "(Optional) Is this item a trap? Something the player doesnt want to get.",
+                    "description": "(Optional) Is this item something the player doesn't want to get? For more information on item classifications, see: https://github.com/ArchipelagoMW/Archipelago/blob/main/docs/world%20api.md#items",
+                    "type": "boolean",
+                    "default": false
+                },
+                "filler": {
+                    "description": "(Optional) Is this item mostly useless and okay to skip placing sometimes? For more information on item classifications, see: https://github.com/ArchipelagoMW/Archipelago/blob/main/docs/world%20api.md#items",
                     "type": "boolean",
                     "default": true
                 },
                 "early": {
                     "description": "(Optional) How many copies of this item are required to be placed somewhere accessible from the start (Sphere 1) \nChoosing 'True' mark all of them to be early",
                     "type": ["boolean", "integer"],
-                    "default": 1
+                    "default": false
                 },
                 "local": {
                     "description": "(Optional) Are all copies of this item supposed to be only in your locations(true) or anywhere(false)",
                     "type": "boolean",
-                    "default": true
+                    "default": false
                 },
                 "local_early": {
-                    "description": "(Optional) How many copies of this item are supposed to be early and only in your locations. \nTrue mark all of them to be local_early. \nCan be used to mark some of the copies of an item to be early and local since 'local' is a toggle between none or all of them.",
+                    "description": "(Optional) How many copies of this item are supposed to be early and only in your locations. \nCan be used to mark some of the copies of an item to be early and local since 'local' is a toggle between none or all of them.",
```

marking this as true is a way to mark all copies as local_early I do agree it could've been in a better order something like
```suggestion
                    "description": "(Optional) How many copies of this item are supposed to be early and only in your locations. \nCan be used to mark some of the copies of an item to be early and local since 'local' is a toggle between none or all of them. \nYou can also set this to 'true'  to have every copy of this item be in local_early",
```

---

Made a similar change that puts the 'true' alternative value closer to the numeric suggested value.